README.md

```diff
@@ -1,13 +1,74 @@
-# Taskmaster [![42](https://i.imgur.com/9NXfcit.jpg)](i.imgur.com/9NXfcit.jpg)
 
-Taskmaster is a 42 project. It is an implementation of a **process supervisor** and aims at getting familiar with job control mechanisms on UNIX-like operating systems.
+<br />
+<p align="center">
+  <a href="https://github.com/pscott/TaskMaster">
+    <img src="assets/logo.png" alt="Logo" width="90" height="90">
+  </a>
 
----
+  <p align="center">
+    <img src="assets/title.png" alt="TaskMaster" width="280" height="70">
+  </p>
+
+  <p align="center">
+    A process control system for UNIX.
+    <br />
+    <a href="https://github.com/pscott/TaskMaster">View Demo</a>
```

Comment:
? This redirects to the same page

Reply:
Just for now. We will add an url tag to usage section later on.
We could even have a free online container having it to run taskmaster later on - if you wish to overkill it.

Reply:
Haha that'd be great, but not sure it's needed indeed. A lil' demo could be a good idea, but I'd go with creating an issue to make sure we don't forget to add it :)